.github/ISSUE_TEMPLATE/01-bug.yml

```
@@ -0,0 +1,60 @@
+name: General Bug
+labels: ["status/needs-triage", "type/bug"]
+description: Report a general problem with Bottlerocket
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This bug report form is for **general issues** with Bottlerocket. 
+        
+        A few tips before submitting a new issue:
+        - Search for similar issues, including [closed issues](https://github.com/bottlerocket-os/bottlerocket/issues?q=is%3Aissue+is%3Aclosed),
+        - Include details about the environment you're running in,
+        - Include any error messages you received, with any required context.
+  - type: dropdown
+    id: variant-family
+    attributes:
+      label: Variant family
+      options:
+        - aws-k8s
+        - aws-k8s (nvidia)
+        - aws-ecs
+        - aws-ecs (nvidia)
+        - vmware-k8s
+        - metal-k8s
+        - Custom variant
+  - type: input
+    id: variant-orchestrator-version
+    attributes:
+      label: Orchestrator version
+      description: For k8s variants, this is your version of Kubernetes. For ECS this is likely "1"
+  - type: textarea
+    id: expect
+    attributes:
+      label: I expected to happen
+      render: bash
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: What actually happened
+      render: bash
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: How to reproduce the problem
+      render: bash
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: About your environment
```

nit: for agreement with the "first person" voice in the other fields

```suggestion
      label: About my environment
```